scipy/ndimage/src/nd_image.c

```diff
@@ -675,15 +675,15 @@ static PyObject *Py_FourierShift(PyObject *obj, PyObject *args)
 static PyObject *Py_SplineFilter1D(PyObject *obj, PyObject *args)
 {
     PyArrayObject *input = NULL, *output = NULL;
-    int axis, order;
+    int axis, order, mode;
 
-    if (!PyArg_ParseTuple(args, "O&iiO&",
+    if (!PyArg_ParseTuple(args, "O&iiiO&",
                           NI_ObjectToInputArray, &input,
-                          &order, &axis,
+                          &order, &axis, &mode,
```

Comment:
Is changing the argument order here a good idea? The function is exported.

Reply:
It is a private module not intended to be used unless it is through one of the Pyhton wrappers, but it doesn't really matter, so moved the new arg to the end.